content/docs/studio/user-guide/prepare-your-repositories.md

```diff
@@ -6,26 +6,15 @@
 
 # Prepare Your Repositories
 
-Iterative Studio creates [projects] by identifying machine learning models,
-datasets, metrics and hyperparameters defined in your Git repositories. These
-values are found in JSON or YAML files in the repository. Additionally, model
-information may be available as Git tags.
-
-New ML model metadata can be added directly from the **Models** tab after
-[creating a project]. You can also use [GTO] or [MLEM].
-
-[projects]: /doc/studio/user-guide/projects-and-experiments/what-is-a-project
-[creating a project]:
-  /doc/studio/user-guide/projects-and-experiments/create-a-project
-[gto]: https://github.com/iterative/gto
-[mlem]: https://mlem.ai/
-
-Datasets, metrics, and hyperparameters can be added to a project in two ways:
-
-1. **Set up DVC repositories**: You can use [DVC](https://dvc.org/) and Git to
-   version your code, data and models all within your Git repositories. Data
-   Version Control, or DVC, is a data and ML experiment management tool that
-   takes advantage of the existing engineering toolset that you're already
+Iterative Studio creates [projects][project] by identifying machine learning
+models, datasets, metrics and hyperparameters defined in your Git repositories.
+These values can be added to a project in two ways:
```

Comment:
Minor: Does the order matter here? I'd always go with "data, params, models, and metrics" if so.

Reply:
Order doesn't matter. I'll change it.

Reply:
Changed.